src/actions/competitors.js

```diff
@@ -0,0 +1,46 @@
+import { database } from './../store';
+import constants from './../components/shared/constants';
+
+//find the head2head score of the 2 competitors
+export const calcHead2Head = (competitorA, competitorB) => {
+  const streak = competitorA.results && competitorA.results[competitorB.uid];
+  let wins = 0;
+  let losses = 0;
+  if (streak) {
```

Comment:
Do you only get wins when you are on a streak? or is it a different kind a streak?

Reply:
This is about the head2head.
hm yeah..naming things. should just call it results